### PR TITLE
Finish doc TODOs

### PR DESCRIPTION
**Pull request by BibMartin**
Last PR for `v0.2` (I hope).

I've created a notebook `Quickstart.ipynb` in examples, and transformed it into `rst` for the docs.

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
 ========
 
-TODO: document here
+You shall take a look at folium's example gallery_.
+
+.. _gallery: http://nbviewer.jupyter.org/github/python-visualization/folium/tree/master/examples/
+
+If this is not enough, you may find fancier examples here_.
+
+.. _here: http://nbviewer.jupyter.org/github/ocefpaf/folium_notebooks/tree/master/
```
Comment (ocefpaf) [inline docs/examples.rst]: I must merge these two into one but I am reluctant to inflate folium's repository with many notebooks.  Maybe a new rep, under `python-visualization` org is in order.

Comment (BibMartin) [inline docs/examples.rst]: I'd say it's okay like this. The first repo contains *tested/guaranteed* notebooks, while the other is more a cookbook.

**Comment by BibMartin** [comment]
Ready and rebased.

@ocefpaf 
> Maybe a new rep, under python-visualization org is in order.

I'm also thinking of another repo for hosting crossfilter, python-nvd3, colormap, etc.

File: docs/examples.rst (hunk 1 above)
Comment (ocefpaf) [inline docs/examples.rst]: Yeah.  That division is important.  Still, I want to get the cookbook away from my personal account and into something more "official," where others will be more likely to contribute.

**Comment by ocefpaf** [comment]
> I'm also thinking of another repo for hosting crossfilter, python-nvd3, colormap, etc.

:+1: 

I saw your `python-nvd3` post BTW.  Awesome stuff!

**Comment by BibMartin** [comment]
> I saw your python-nvd3 post BTW. Awesome stuff!

Thanks. I think smething can be done to get `python-nvd3`, `folium` and `vincent` in a common mindset.

**Pull request merged**